sdk/eventhub/azure-eventhub/azure/eventhub/aio/_client_base_async.py

```diff
@@ -421,8 +420,8 @@ async def _get_partition_properties_async(
         response = await self._management_request_async(
             mgmt_msg, op_type=MGMT_PARTITION_OPERATION
         )
-        partition_info = response.value  # type: Dict[bytes, Union[bytes, int]]
-        output = {}  # type: Dict[str, Any]
+        partition_info: Dict[bytes, Union[bytes, int]] = response.value
```

Comment:
on the sync side, `partition_info` is  `Dict[bytes, Any]`. I think bytes, int might be better suited